tests/fullydistributed_grids/memory_consumption_01.mpirun=5.output

```diff
@@ -0,0 +1,14 @@
+
+DEAL:0:2d::93274
+
+DEAL:1:2d::121820
```

Comment:
We have made the experience that the output of the memory consumption as a plain number is a pain to maintain because the result depends on the number type (32 vs 64 bit integers), changes somewhere in the code that lead to different allocated sizes, or also different implementations of the STL library in terms of how to dynamically grow vectors. Can we find a way to approximately control this number, say, by checking that it is in the right range (and assert that this is the case at least for 32 and 64 bit builds)?